tests/autoyast/installation.pm

```diff
@@ -330,13 +330,13 @@ sub run {
 
     # Cannot verify second stage properly on s390x, so reconnect to already installed system
     if (check_var('ARCH', 's390x')) {
-        reconnect_mgmt_console(timeout => 500, grub_timeout => 180);
+        reconnect_mgmt_console(timeout => 700, grub_timeout => 180);
         return;
     }
     # For powerVM need to switch to mgmt console to handle the reboot properly
     if (is_pvm()) {
         prepare_system_shutdown;
-        reconnect_mgmt_console(timeout => 500);
+        reconnect_mgmt_console(timeout => 700);
```

Comment:
PowerVM is pretty fast arch, is the other Power the slow one, but I cannot see any failure, we can just keep the fix for s390x, wdyt?

Reply:
Ah, that's a typo, I think I got confused.